crates/devolutions-pedm-shared/src/policy.rs

```diff
@@ -311,23 +311,17 @@ pub trait Identifiable {
 #[derive(Serialize, Deserialize, JsonSchema, Debug, Clone, Hash, PartialEq, Eq)]
 #[serde(rename_all = "PascalCase")]
 pub struct Profile {
-    pub id: Uuid,
+    pub id: Option<i64>,
```

Comment:
question: So we ended up with changing for `i64`? Is it really okay for it to be optional?

Reply:
Good catch, this was a leftover from refactoring.

I changed the IDs to i64 as it's nicer for the database. The ID start at "1", so "0" is used for "no ID". 

Originally I had this as `Option` because I thought it was cleaner, but actually OpenAPI doesn't generate a `long?` on the client, it simply sets the data contract to **not serialize** the default value. So in the end there was absolutely no benefit to using `Option` here, and it causes noise on the rust side. I simply forgot to remove the `Option` after figuring all that out :) 

Removed now, which makes the API cleaner, which I'm happy about. Thanks!